client/reader/recommendations/index.js

```diff
@@ -1,44 +0,0 @@
-/** @format */
-/**
- * External dependencies
- */
-import page from 'page';
-import { forEach } from 'lodash';
-
-/**
- * Internal dependencies
- */
-import { recommendedPosts } from './controller';
-import { preloadReaderBundle, sidebar, updateLastRoute } from 'reader/controller';
-import config from 'config';
-
-export default function() {
-	// Cold Start no longer exists - redirect to /
-	page( '/recommendations/start', '/' );
-
-	// Blog Recommendations no longer exists as its own page - redirect to /
-	page( '/recommendations', '/read/search' );
```

Comment:
It'd be nice to keep this redirect. Can we add it to `legacyRedirects` in the main Reader controller perhaps?

Reply:
Do we still link to there from anywhere?
I figured it made sense as a grace period, but I don't see a point to keeping around legacy redirects forever within client code

Reply:
I'd still leave it for now.